lib/muser.h

```diff
@@ -226,9 +226,9 @@ typedef void (lm_log_fn_t) (void *pvt, const char *const msg);
  *
  * @returns the number of bytes read or written
  */
-typedef ssize_t (lm_cap_access_t) (void *pvt, const uint8_t id,
-                                   char * const buf, size_t count,
-                                   loff_t offset, const bool is_write);
+typedef ssize_t (lm_cap_access_t) (void *pvt, uint8_t id,
+                                   char *buf, size_t count,
+                                   loff_t offset, bool is_write);
```

Comment:
Looks like this can be done in 3 lines, considering the 80C width.
e.g:
typedef ssize_t (lm_cap_access_t) (void *pvt, uint8_t id, char *buf,
                                                            size_t count, loff_t offset, bool is_write);

Reply:
Do you mean 2 lines? Yeah, it could. I slightly prefer 3 to keep related variables grouped together (eg. buf and count). I think some of these will need to be further review wrt the usage of const. Let me know if you think I should pack it now or later and I can resend.

Reply:
Yes I mean 2 lines. But its fine, you can also fix this later.